e2e/pages/events/enterAllocationDecisionEvent.page.js

```diff
@@ -0,0 +1,20 @@
+const I = actor();
+
+module.exports = {
+
+  fields: {
+    allocationDecisionRadioGroup: '#allocationDecision_proposal',
+    proposalReason: '#allocationDecision_proposalReason',
+  },
+
+  selectAllocationDecision(proposal) {
+    I.waitForElement(this.fields.allocationDecisionRadioGroup);
+    within(this.fields.allocationDecisionRadioGroup, () => {
+      I.click(locate('label').withText(proposal));
```

Comment:
Hmmm slightly concerned about how fragile this could be. What if CCD break ID / For functionality, can we not select radio button directly?

Reply:
It actually doesn't work for me (I remember you mentioned having some issues as well) I tried right now:
     Clickable element "#allocationDecision_proposal-Lay justices" was not found by text|CSS|XPath

maybe some issues with the spaces? I tried without "#" and it didn't work.. do you know what might be the reason?

Reply:
My issue was with checkboxes and i'd to use `c6: locate('input').withAttr({id: 'proceedingTypes-NOTICE_OF_PROCEEDINGS_FOR_PARTIES'})`. It would make sense if it was an issue with the space though. I'm assuming the space is coming from the list element code, maybe you could make the fixed lists listElementCode more enum like?

Reply:
Actually no, this is already merged :/ changing listElementCode would break ES. For now let's just leave this agreed?

Reply:
Yes, it's all fixed list from allocationProposal so not even recent story :( I will leave it for now. Thanks though, it's definitely good point.

Reply:
Juggled the code a bit, please check.